examples/inference/hybrid_llama.py

```diff
@@ -0,0 +1,96 @@
+import argparse
+import time
+
+import pytest
+import torch
+import torch.distributed as dist
+import transformers
+from transformers import LlamaForCausalLM, LlamaTokenizer
+
+import colossalai
+from colossalai.inference import CaiInferEngine, LlamaModelInferPolicy
+from colossalai.testing import clear_cache_before_run, rerun_if_address_is_in_use, spawn
+
+
+def pipeline_inference_test(args):
+    llama_model_path = args.path
+    max_input_len = args.max_input_len
+    max_output_len = args.max_output_len
+    max_batch_size = args.batch_size
+    micro_batch_size = args.micro_batch_size
+    tp_size = args.tp_size
+    pp_size = args.pp_size
+    rank = dist.get_rank()
+
+    tokenizer = LlamaTokenizer.from_pretrained(llama_model_path)
+    tokenizer.pad_token_id = tokenizer.unk_token_id
+    model = LlamaForCausalLM.from_pretrained(llama_model_path, pad_token_id=tokenizer.eos_token_id)
```

Comment:
can this support colossal-llama2?  https://github.com/hpcaitech/ColossalAI/blob/main/examples/inference/colossal_llama2_demo.py

Reply:
Yes. Llama and Llama2 is same except the num_key_value_heads.

Reply:
tested it?

Reply:
TODO: Later